test/resources/awss3_jl_test.yaml

```diff
@@ -236,6 +236,38 @@ Resources:
               - s3:DeleteObjectVersion
             Resource: !Sub arn:aws:s3:::${BucketPrefix}*/*
 
+  DeleteAllVersionsTestsetRole:
```

Comment:
@omus I am _very_ unsure if this is sufficient to add new test permissions

Reply:
This is perfect. If we wanted to go to the extreme we could use a more restrictive permissions set when testing just `s3_delete_all_versions` but I think this is fine